autoscale_cloudroast/test_repo/autoscale/system/integration/test_system_integration_rcv3.py

```diff
@@ -650,6 +665,121 @@ def test_scale_up_down_on_rcv3_and_clb(self):
                           'count of {1}'.format(num_clb_nodes_after_scale,
                                                 self.min_servers))
 
+    @tags(speed='slow', type='rcv3', rcv3_mimic='pass', working='true')
```

Comment:
Non-blocking: how do you plan to use `working`? I think using `skip` to specify non-working tests is better option.